opencti-platform/opencti-front/src/private/components/data/tasks/TasksList.jsx

```diff
@@ -373,12 +374,14 @@ class TasksListComponent extends Component {
                       </Typography>
                       {nsdt(task.last_execution_date)}
                     </Grid>
-                    <Grid item={true} xs={2}>
-                      <Typography variant="h3" gutterBottom={true}>
-                        {t('Scope')}
-                      </Typography>
-                      <TaskScope scope={task.scope} label={t(task.scope)} />
-                    </Grid>
+                    {computeLabel(task.type, task.scope)
+                      && <Grid item={true} xs={2}>
+                        <Typography variant="h3" gutterBottom={true}>
+                          {t('Scope')}
+                        </Typography>
+                        <TaskScope scope={task.scope} label={t(computeLabel(task.type, task.scope))} />
```

Comment:
can't we just do something like: t(task.scope ?? task.type) instead of creating a function just for that?

Reply:
I prefer to separate the logic from the purely UI part. But, we can discuss it.